ygen/enumgen.go

```diff
@@ -882,9 +898,9 @@ func (s *enumGenState) resolveTypedefEnumeratedName(e *yang.Entry, noUnderscores
 	// as this is unique, whereas post-camelisation, we may have name clashes. Since a typedef
 	// does not have a 'path' in Goyang, we synthesise one using the form
 	// defining-module-name/typedef-name.
-	nodeForModuleName := e.Node
+	nodeForModuleName := args.contextEntry.Node
 	if useDefiningModuleForTypedefEnumNames {
-		nodeForModuleName = enumType.Base
+		nodeForModuleName = args.subsumingType.Base
```

Comment:
Can `args.subsumingType` ever be `nil` here?

Reply:
Added error handling.